packages/mermaid/src/diagrams/treeView/styles.ts

```diff
@@ -0,0 +1,28 @@
+import type { DiagramStylesProvider } from '../../diagram-api/types.js';
+import { cleanAndMerge } from '../../utils.js';
+import type { TreeViewDiagramStyles } from './types.js';
+
+const defaultTreeViewDiagramStyles: Required<TreeViewDiagramStyles> = {
+  fontSize: '16px',
+  lineColor: 'black',
+  fontFamily: 'trebuchet ms",verdana,arial,sans-serif',
```

Comment:
trebuchet ms is Microsoft's we prefer to use open source fonts.

Reply:
On second thought, I feel like inheriting the global font is probably better so I removed the option.